com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs

```diff
@@ -301,7 +301,7 @@ void UpdateAbcTree(aiObject top, AlembicTreeNode node, double time, bool createM
 
             if (!serializeMesh)
             {
-                foreach (var meshFilter in node.gameObject.GetComponentsInChildren<MeshFilter>())
+                foreach (var meshFilter in node.gameObject.GetComponentsInChildren<MeshFilter>(includeInactive: true))
```

Comment:
I don't think this will entirely work with every cases. If you manually setup an alembic stream with a file that is outside of the project, meshes will still be generated with the wrong flag, `serializeMesh = true`.

`AlembicStreamPlayer.cs` where the initialization is triggered when you load from file:
```
bool InitializeAfterLoad()
{
    var ret = LoadStream(true, true);
    ...
}
```

Reply:
NVM, I mistook the fix for something else.